docs/platforms/emr_serverless.md

```diff
@@ -179,6 +179,202 @@ This defines an asset that runs a spark job on an EMR Serverless Application def
+
+<span style="color:#4E9A06"><b>✓ Successfully validated 1 assets across 1 pipeline, all good.</b></span>
+
+<b>Starting the pipeline execution...</b>
+<span style="color:#8D8F8A">[2025-05-11 18:10:00]</span> <span style="color:#C4A000">Running:  users</span>
+<span style="color:#8D8F8A">[2025-05-11 18:10:00]</span> <span style="color:#C4A000">... output omitted for brevity ...</span>
+<span style="color:#8D8F8A">[2025-05-11 18:10:05]</span> <span style="color:#C4A000">Finished: users</span>
+
+<span style="color:#4E9A06"><b>Executed 1 tasks in 5s</b></span>
+</pre>
+:::
+
+#### Enabling quality checks
+
+In order to run quality checks, we need to:
+1. Create an Athena table in our AWS account.
+2. Configure an `athena` connection in our `bruin.yml` file.
+
+To start, let's first create our Athena table. Go to your AWS Athena console and run the following DDL Query to create a table over the data our pyspark job created.
+```sql
+CREATE EXTERNAL TABLE users (id int, name string, age int)
+ROW FORMAT DELIMITED
+FIELDS TERMINATED BY ','
+LINES TERMINATED BY '\n'
+STORED AS TEXTFILE
+LOCATION 's3://acme/user/list' 
+```
+
+> [!TIP]
+> For more information on creating Athena tables, see [Create tables in Athena](https://docs.aws.amazon.com/athena/latest/ug/creating-tables.html) and [Use SerDe](https://docs.aws.amazon.com/athena/latest/ug/serde-reference.html) in the AWS Athena Documentation.
+
+Next, update your `bruin.yml` file with an athena connection.
+
+```yaml [bruin.yml]
+environments:
+  default:
+    connections:
+      emr_serverless:
+      - name: emr_serverless-default
+        access_key: AWS_ACCESS_KEY_ID
+        secret_key: AWS_SECRET_ACCESS_KEY
+        region: eu-north-1
+        application_id: EMR_APPLICATION_ID
+        execution_role: IAM_ROLE_ARN
+        workspace: s3://acme/bruin-pyspark-workspace/
+      athena:                                           # [!code ++]
+      - name: quality-tests                             # [!code ++]
+        access_key_id: AWS_ACCESS_KEY_ID                # [!code ++]
+        secret_access_key: AWS_SECRET_ACCESS_KEY        # [!code ++]
+        region: eu-north-1                              # [!code ++]
+        query_results_path: "s3://acme/athena-output/"  # [!code ++]
+
+```
+
+Now we can update our assets to define some quality checks. For this example, we're going to add one column and one custom check.
+
+::: code-group
+```bruin-python [quality-checks-example/assets/users.py]
+""" @bruin
+name: users
+type: emr_serverless.pyspark
+columns:              # [!code ++]
+  - name: id          # [!code ++]
+    type: integer     # [!code ++]
+    checks:           # [!code ++]
+      - non_negative  # [!code ++]
+custom_checks: # [!code ++] 
+  - name: users are adults # [!code ++]
+    query: SELECT count(*) from users where age < 18 # [!code ++]
+    value: 0 # [!code ++]
+parameters: # [!code ++]
+  athena_connection: quality-tests # [!code ++]
```

Comment:
I suggest setting up the examples in such a way that they set this in the `pipeline.yml` by default.

Reply:
I felt that the example has too many pieces already, so I didn't want to add to that.

Instead, I've added a tip for using default values.